deploy.sh

```diff
@@ -32,5 +32,6 @@ sam deploy --stack-name "$stack_name" \
   --parameter-overrides \
   Environment=dev \
   SecretPrefix=pre-merge-test \
+  LambdaDeploymentPreference=AllAtOnce\
```

Comment:
is this required? I am not sure if this would override values set in the pipeline

Reply:
It gives the option to change deployment strategy in dev when deploying with the script - I used it for testing the different options but no not really required, have removed